pkgs/development/tools/misc/igprof/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchFromGitHub, cmake}:
+
+stdenv.mkDerivation rec {
+  version = "5.9.16";
+  name = "igprof-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "igprof";
+    repo   = "igprof";
+    rev    = "v${version}";
+    sha256 = "0glpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  #makeFlags = ''CFLAGS="$CFLAGS -fPIC -Wno-error=unused-result"'';
+
+  meta = {
+    description = "The Ignominous Profiler";
+
+    longDescription = ''
+      IgProf is a simple nice tool for measuring and analysing application
+      memory and performance characteristics. IgProf requires no changes to the
+      application or the build process.
+    '';
+
+    license = stdenv.lib.licenses.gpl2;
+
+    homepage = https://igprof.org/;
+    platforms = stdenv.lib.platforms.linux;
+  };
```

Comment:
(optional) Any reason you didn't add yourself as maintainer? It's IIRC not required, but I'd prefer to merge packages with maintainers listed in the meta section :)